multiversx_sdk/abi/abi_definition.py

```diff
@@ -86,14 +86,16 @@ def __init__(self,
                  inputs: List["ParameterDefinition"],
                  outputs: List["ParameterDefinition"],
                  payable_in_tokens: List[str],
-                 only_owner: bool) -> None:
+                 only_owner: bool,
+                 title: str) -> None:
```

Comment:
Tiny breaking chance, but should not be noticed by developers.

The alternative would have been to use `Optional[str]`. 

But it's all right this way, as well.

Reply:
I updated to use the Optional[str] :+1: